server/pulp/plugins/rsync/publish.py

```diff
@@ -258,6 +258,36 @@ def process_main(self):
             raise PulpCodedException(message=output)
 
 
+class UpdateLastPredistDateStep(PublishStep):
```

Comment:
So is it likely that the only plugin that will need this code is the RPM plugin?